server/db/postgres/sequelize_config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  development: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_development',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  test: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_test',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  production: {
+    use_env_variable: 'POSTGRES_DB_URL'
+  }
```

Comment:
It looks like sequelize figures out the dialect from the url, since that is of the form:

```
dialect://asdf:ghjk@whatever:1234/qwerty
```

You can see it pulls out the 'postgres' part [here](https://github.com/sequelize/sequelize/blob/master/lib/sequelize.js#L132)